.gitignore

```diff
@@ -1,4 +1,4 @@
-/vendor/
+vendor/
```

Comment:
The whole folder `sample` is not exported with composer. In fact only the following items will be exported
- `src` folder
- `composer.json`
- `LICENSE`
- `NOTICE`
- `README.md`